certificate-manager/byo-root.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "Smallstep Certificate Manager: Bring Your Own Root"
+html_title: "Smallstep Certificate Manager: Bring Your Own Root"
+description: Use your existing root to sign an intermediate certificate for use in Smallstep
+---
+
+
+With the Smallstep platform, you can bring your existing PKI to us, by signing a Smallstep intermediate CA from your existing root.
+
+Intermediate CAs (also called subordinate CAs) are used to sign and issue leaf certificates to subscribers. Intermediates aren't generally included in trust stores, making them easier to revoke and rotate. This is why they are commonly used for online CAs like Smallstep Certificate Manager.
+
+This tutorial will walk you through importing your root CA certificate to Smallstep, and signing an intermediate CA that we'll use to issue certificates on your behalf.
+
+The steps are:
+1. Upload your root CA certificate to Smallstep.
+2. Customize your intermediate CA certificate properties. Smallstep will create an intermediate private key and a Certificate Signing Request (CSR) for you.
+3. Download the CSR and use your root CA to sign in
```

Comment:
Not a biggie, but you started with periods at the end of each sentence and ended w/o by step 3